src/m365/spo/base-permissions.spec.ts

```diff
@@ -162,4 +162,41 @@ describe('BasePermissions', () => {
 
     assert.strictEqual(basePermissions.has(PermissionKind.FullMask), true);
   });
+
+  it('parse returns correct list of permissions kind', () => {
+    
```

Comment:
Let's remove this unnecessary empty line

Reply:
✅